spec/lib/os_places_api/client_spec.rb

```diff
@@ -85,7 +134,21 @@
           {
             "average_latitude" => average_latitude,
             "average_longitude" => average_longitude,
-            "results" => [location].as_json,
+            "results" => locations.as_json,
+          },
+        )
+      end
+
+      it "should query OS Places API and return unique results" do
```

Comment:
Nitpick - this reads more clearly to me:

```suggestion
      it "should filter out duplicate `UPRN` results from the OS Places API response" do
```